### DISCUSSION TITLE
Can we start publishing the rust bindings to crates.io?

### DISCUSSION THREAD

**Discussion by smarter** [tree-sitter/tree-sitter-scala#191] [General]
Once https://github.com/tree-sitter/tree-sitter-scala/pull/190 is merged, we'll be able to compile the Rust bindings. It'd be great to then publish them so they can be used in downstream project such as https://github.com/github/stack-graphs/ (which github is apparently starting to use for go-to-definition/find-all-references: https://docs.github.com/en/repositories/working-with-files/using-files/navigating-code-on-github#precise-and-search-based-navigation)

**Comment by eed3si9n** [comment]
Closing the loop here we've been publishing to crates.io since https://github.com/tree-sitter/tree-sitter-scala/releases/tag/v0.20.1
